test/plots/plotTests.ts

```diff
@@ -1,584 +1,302 @@
 
-      let svg = TestMethods.generateSVG();
-      plot.renderTo(svg);
+      const oldDomain = scale.domain();
 
-      assert.deepEqual(categoryScale.domain(), ["B", "A"], "extent is in the right order");
-      svg.remove();
-    });
+      const svg = TestMethods.generateSVG();
+      plot.anchor(svg);
 
-    it("animated() getter", () => {
-      let plot = new Plottable.Plot();
-      assert.strictEqual(plot.animated(), false, "by default the plot is not animated");
-      assert.strictEqual(plot.animated(true), plot, "toggling animation returns the plot");
-      assert.strictEqual(plot.animated(), true, "animated toggled on");
-      plot.animated(false);
-      assert.strictEqual(plot.animated(), false, "animated toggled off");
+      plot.detach();
+      assert.deepEqual(scale.domain(), oldDomain, "detaching the plot removes its extents from the scale");
+      svg.remove();
     });
 
     describe("clipPath", () => {
       it("uses the correct clipPath", () => {
-        let svg = TestMethods.generateSVG();
-        let plot = new Plottable.Plot();
+        const svg = TestMethods.generateSVG();
+        const plot = new Plottable.Plot();
         plot.renderTo(svg);
         TestMethods.verifyClipPath(plot);
         svg.remove();
       });
 
-      it("updates the clipPath reference when render()-ed", () => {
-        if (window.history == null || window.history.replaceState == null) { // not supported on IE9 (http://caniuse.com/#feat=history)
-          return;
-        }
+      // not supported on IE9 (http://caniuse.com/#feat=history)
+      if (window.history != null && window.history.replaceState != null) {
+        it("updates the clipPath reference when rendered", () => {
+          const svg = TestMethods.generateSVG();
+          const plot = new Plottable.Plot();
+          plot.renderTo(svg);
+
+          const originalState = window.history.state;
+          const originalTitle = document.title;
+          const originalLocation = document.location.href;
+          window.history.replaceState(null, null, "clipPathTest");
+          plot.render();
+
+          const clipPathId = (<any> plot)._boxContainer[0][0].firstChild.id;
+          const expectedPrefix = (/MSIE [5-9]/.test(navigator.userAgent) ? "" : document.location.href).replace(/#.*/g, "");
+          const expectedClipPathURL = "url(" + expectedPrefix + "#" + clipPathId + ")";
+
+          window.history.replaceState(originalState, originalTitle, originalLocation);
+
+          const normalizeClipPath = (s: string) => s.replace(/"/g, "");
+          assert.strictEqual(normalizeClipPath((<any> plot)._element.attr("clip-path")), expectedClipPathURL,
+            "the clipPath reference was updated");
+          svg.remove();
+        });
+      }
+    });
 
-        let svg = TestMethods.generateSVG();
-        let plot = new Plottable.Plot();
-        plot.renderTo(svg);
+    describe("setting the attribute of plot elements", () => {
```

Comment:
You could label this `"setting attributes with attr()"` if you want, since parens in the name of a `describe()` can be found by `grep`.